README.md

```diff
@@ -1,61 +1,95 @@
 # FeedHenry Custom LDAP Connector
+
 [![Dependency Status](https://img.shields.io/david/feedhenry-templates/fh-connector-ldap-cloud.svg?style=flat-square)](https://david-dm.org/feedhenry-templates/fh-connector-ldap-cloud)
 
-The FeedHenry Custom LDAP Connector.
+* [The FeedHenry Custom LDAP Connector.](#the-feedhenry-custom-ldap-connector)
+  + [Required Environment Variables](#required-environment-variables)
+  + [AUTH](#auth)
+* [Tests](#tests)
+  + [Unit tests](#unit-tests)
+  + [Unit coverage](#unit-coverage)
+
+## The FeedHenry Custom LDAP Connector.
 
-# Required Environment Variables
+### Required Environment Variables
 
 * LDAP_URL : The URL of the LDAP server (including the ldap protocol). E.g. `ldap://testing.feedhenry.com`
 * DN_PREFIX: A prefix to be added before the username. E.g. `CN=`
 * DN: A suffix to be added after the user name. E.g. `,CN=Roles,CN=LDAPtest,DC=feedhenry,DC=com`
 
-# LDAP Connector API
+### AUTH 
+
+Validate a user against a LDAP server
 
-# AUTH [/cloud/auth]
+|              |              |
+|--------------|--------------|
+| Endpoint     | /cloud/auth  |
+| HTTP Method  | POST         |
 
-Bind against a LDAP server
 
-## AUTH [POST]
+#### Request (application/json)
 
-Validate a user against a LDAP server
+##### Body
+
+```json
+{
+  "username": "users username",
+  "password": "users password"
+}
+```
+
+#### Response 200 (application/json)
+
+##### Body
+
+```json
+{
+  "status": "ok",
+  "message": "Successfully Authenticated"
+}
+```
+
+#### Response 401 (application/json)
+
+##### Body
 
-+ Request (application/json)
+```json
+{
+  "status": "unauthorised",
+  "message": {
+    "dn": ""
+    "code": Error Code
+    "name": "The authorisation error name"
+    "message": "The authorisation error message"
+  }
+}
+```
 
-    + Body
+#### Response 500 (application/json)
 
-            {
-              "username": "users username",
-              "password": "users password"
-            }
+##### Body
 
-+ Response 200 (application/json)
+```json
+{
+  "status": "error",
+  "message": "You need to provide a username and password."
+}
+```
 
-    + Body
+## Tests
 
-            {
-              "status": "ok",
-              "message": "Successfully Authenticated"
-            }
+All the tests are in the "test/" directory. The cloud app is using mocha as the test runner. 
 
-+ Response 401 (application/json)
+### Unit tests
```

Comment:
Same as other PR, just add that mongod needs to be running locally and run `grunt serve` before running the tests